editor/editor_file_system.cpp

```diff
@@ -1411,6 +1411,9 @@ String EditorFileSystem::_get_global_script_class(const String &p_type, const St
 }
 
 void EditorFileSystem::_scan_script_classes(EditorFileSystemDirectory *p_dir) {
+	if (p_dir->parent && p_dir->parent->name == "addons" && EditorNode::get_singleton()->is_addon_plugin_enabled(p_dir->name)) {
```

Comment:
You said you're filtering out inactive plugins, but doesn't this filter out *enabled8 plugins?

Reply:
@akien-mga Whoops. You're right. It was previously two separate if statements (when I tested it), and when I refactored it to one I forgot the exclamation point. Nice catch. Fixed now.